src/shared/common/components/summary/actions/action-summary-factory.jsx

```diff
@@ -10,10 +11,11 @@ function renderComponent(Component, items) {
 }
 
 const getActionSummary = (actionType, actions) => {
-
     switch (actionType) {
         case 'Appeals':
             return renderComponent(AppealSummary, actions.find(actions => actions.title === actionType));
+        case 'Recordinterest':
```

Comment:
Might be an idea asking @gary-newbolt about this - I'm not sure I know what the reason would be from the top of my head for this. I agree this would probably be nicer pascal cased.

Reply:
That was probably missed on the original code review for the creation of the `RecordInterest` action... it's worth changing for sure, you would probably need to make a corresponding change in `hocs-casework` off of the top of my head, it's worth a check

Reply:
In the database it appears to be `recordInterest` and we then change to `Recordinterest`. I think its just a change to the way the action type is handled in the frontend. If you need a hand with this @JosephRogerson give me a shout!

Reply:
I've changed this now so that we compare against the value that is passed to the frontend from the database rather than the formatted value